src/store/state.ts

```diff
@@ -26,5 +29,6 @@ export const state: GeneralState = {
   currentNetworkIdx: 0,
   currentAccountIdx: 0,
   currentCustomEndpoint: '',
+  currentTheme: 'DARK',
```

Comment:
I set the default theme as dark, should I switch it back to light?

Reply:
It would be better if you can make the default theme as the media setting (client system default). That can be done in the next PR